index.html

```diff
@@ -6,8 +6,14 @@
 </head>
 <body>
 <table border="1" style="width:100%; height:900px;">
-	<tr><td colspan="2">header</td></tr>
 	<tr>
+			<td colspan="2" bgcolor="#99FF99"><img
+				src="https://dl.dropboxusercontent.com/u/74824888/e2f75e044948eee49a39a1eb8e5c0ce1.jpg"

```

Comment:
Pictures better to have in a folder in the project. Create a separate folder and store images there.